code/src/components/Header/Hero.css

```diff
@@ -0,0 +1,118 @@
+.Background {
+  display: flex;
+  flex-wrap: nowrap;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 0;
+  width: 100%;
+  height: 100vh;
+}
+
+.hero-img {
+    flex: 1;
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+  }
+  
+  .block-color {
+    flex: 0.45;
+    height: 100%;
+    background-color: var(--primary-color);
+    opacity: 1;
+  }
+  
+
+  @media (min-width: 641px) {
+    .Background {
+      width: 100%;
+      height: 100vh;
+    }
+    .hero-img {
+      background-position: 65% 30%;
+    }
+
+    .block-color {
+      flex: 0.45;
+    }
+  }
+
+  @media (min-width: 1025px) {
+    .Background {
+      width: 100%;
+      height: 100vh;
+    }
+
+    .hero-img {
+      background-position: 65% 25%;
+    }
+
+    .block-color {
+      flex: 0.35;
+    }
+  }
+
+/*
+.Background {
+  display: flex;
+  flex-wrap: nowrap;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 0;
+  width: 100%;
+  height: 100%;
```

Comment:
The height: 100%; property in the .hero-img class can be replaced with height: 100vh; to ensure the image covers the entire viewport height.